app/src/main/java/org/thoughtcrime/securesms/permissions/Permissions.java

```diff
@@ -143,7 +143,7 @@ public void execute() {
 
       if (!isInTargetSDKRange || permissionObject.hasAll(requestedPermissions)) {
         executePreGrantedPermissionsRequest(request);
-      } else if (rationaleDialogMessage != null && rationalDialogHeader != null) {
+      } else if (rationaleDialogMessage != null) {
```

Comment:
Is this safe?

Reply:
@SessionHero01 I modified the rationale dialog to now be able to display _without_ a header, since we now have designs that require it. You can see in the RationaleDialog I added some code to display a generic title when no icons are provided for the header.